jupyter-notebooks/orders_basemaps/SDK_order_basemaps.ipynb

```diff
@@ -0,0 +1,403 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Order a Basemap Using the Orders API & SDK via AOI & Quad ID"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview ##\n",
```

Comment:
In this overview section, I recommend providing a very high level outline for the reader of what this notebook is and what they can expect. Something like:

"In this notebook, you will learn how to order a basemap using your Area of Interest (AOI) and a Quad ID. We will place this order via Planet's Orders API using our Planet Python SDK."

^ When you write a description like the above, make sure you're linking to relevant materials on our DevCenter. Imagine the person reading this doesn't know what a Quad ID is or what a basemap is. 

Then, I'd give a high-level overview of the steps involved:
- Identify the basemap to order
- Identify Mosaic and Quad IDs for your basemap
- Build your order packet
- Create and place your basemaps order

These sections will give the reader a general idea of what they'll do and will help anchor them to the content of the notebook oce they get started. You can also use the steps you outline here as headings for each section, to make sure the reader knows what stage they are at at any given time so they don't get lost in the details of it all